### PR TITLE
add default fee tokens if not configured

### LINKED ISSUE #142: Fee Token defaults
Right now, owner must define FEE_TOKEN in order to start the node.

Let's have some defaults:
 - if FEE_AMOUNT is not defined, make it 0
 - if FEE_TOKEN is not defined, make it equal to OCEAN address per each chain
 
 This will ease onboarding for non tech users

### PR DESCRIPTION
**Pull request by paulo-ocean**
Fixes #142 

Changes proposed in this PR:

- always have default values for FEE_TOKENS and FEE_AMOUNT, even if not configured
- refactor some related code
- common util function for getting artifacts addresses
- some related cleaning on the env. vars + logging

### CODE REVIEW

File: src/utils/address.ts [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+        `${homedir}/.ocean/ocean-contracts/artifacts/address.json`,
+      'utf8'
+    )
+    return JSON.parse(data)
+  } catch (error) {
+    return null
+  }
+}
+// default Ocean token (only) addresses per chain
+export const OCEAN_TOKEN_ADDRESS_PER_CHAIN = {
```
Comment (alexcos20) [inline src/utils/address.ts:22]: no need for hardcoding, you can find them in address.json, see https://github.com/oceanprotocol/contracts/blob/main/addresses/address.json

**Review by alexcos20 — CHANGES_REQUESTED** [review]

File: src/utils/address.ts (hunk 1 above)
Comment (paulo-ocean) [reply src/utils/address.ts:22]: yeah, i missed that, we are already importing it in other place
thanks

File: src/components/core/feesHandler.ts [hunk 2]
```diff
@@ -343,28 +347,28 @@ export async function getFees(task: GetFeesCommand): Promise<P2PCommandResponse>
...
  * @returns the token address
  */
-export function getProviderFeeToken(chainId: string): string | null {
-  const result = config.feeStrategy.feeTokens.filter(
-    (token: FeeTokens) => token.chain === chainId
+export async function getProviderFeeToken(chainId: number): Promise<string> {
+  const result = (await getConfiguration()).feeStrategy.feeTokens.filter(
+    (token: FeeTokens) => Number(token.chain) === chainId
   )
   return result.length ? result[0].token : null
```
Comment (alexcos20) [inline src/components/core/feesHandler.ts:373]: you should return zero address (0x0000000000000000000000000000000000000000) instead of null

Comment (paulo-ocean) [reply src/components/core/feesHandler.ts]: alright, thanks. just updated it

**Review by bogdanfazakas — APPROVED** [review]

**Review by alexcos20 — APPROVED** [review]
lgtm

**Pull request merged**